kata-webhook/README.md

```diff
@@ -30,26 +30,9 @@ and sample admission controller we created by running
 ```bash
 $ kubectl apply -f deploy/
 ```
-The admission webhook (deploy/webhook-registration.yaml)
-is setup to exclude certian namespaces from being run with Kata using filters on namespace labels.
 
-```yaml
-    namespaceSelector:
-      matchExpressions:
-        -  {key: "kata", operator: NotIn, values: ["false"]}
-```
-The rook operators for example are marked as such.
-
-```yaml
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: rook-ceph-system
-  labels:
-    kata: "false"
-```
-
-Pods not explicitly excluded by the namespace filter are dynamically tagged to run with Kata with some exceptions.
+Pods are dynamically tagged to run with Kata based policies setup with the webhook.
```

Comment:
Can I check the context of this paragraph is still correct? In my head it used to read for instance 'we exclude rook', and now I read it as 'we include rook' ?

Reply:
Now that I read it I do not quite understand what I was trying to say.

In short with this the webhook can be setup to ignore pods based on the namespace, net==host or some other property that we choose to filter on. All other pods will run with Kata. Let me reword this to make it clearer.